src/js/types.d.ts

```diff
@@ -0,0 +1,169 @@
+import { QuickStartCatalogPage } from '@patternfly/quickstarts';
+/**
+ * @deprecated
+ * Only use as placeholder
+ */
+export type AnyObject = {
+  [key: string]: any;
+};
+
+/**
+ * TODO: Move to the component once it is migrated to TS
+ */
+export type NavItem = {
+  href?: string;
+  routes?: NavItem[];
+  navItems?: NavItem[];
+  active?: boolean;
+};
+
+export type ChromeUser = {
+  entitlements: {
+    [key: string]: {
+      is_entitled: boolean;
+      is_trial: boolean;
+    };
+  };
+  identity: {
+    account_number: string;
+    internal: {
+      org_id: string;
+      account_id: string;
+    };
+    type: string;
+    user?: {
+      username: string;
+      email: string;
+      first_name: string;
+      last_name: string;
+      is_active: boolean;
+      is_internal: boolean;
+      is_org_admin: boolean;
+      locale: string;
+    };
+  };
+};
+
+export type VisibilityFunctions = {
+  isOrgAdmin: () => Promise<boolean>;
+  isActive: () => Promise<boolean>;
+  isInternal: () => Promise<boolean>;
+  isEntitled: () => Promise<boolean>;
+  isProd: () => boolean;
+  isBeta: () => boolean;
+  isHidden: () => true;
+  withemai: (toHave: string) => Promise<boolean>;
```

Comment:
This should be `withEmail` and `toHave` should be unknon amount of attrs.
```suggestion
  withEmail: (...toHave: string[]) => Promise<boolean>;
```

Reply:
Type is now imported from the types package.